packages/perseus/src/widgets/__tests__/radio/base-radio_test.jsx

```diff
@@ -10,65 +10,94 @@ import BaseRadio from "../../radio/base-radio.jsx";
 
 import type {APIOptions} from "../../../types.js";
 
+function generateChoice(options) {
```

Comment:
**suggestion (non-blocking):** Even though this function isn't being exported, it can be helpful to add types this function to narrow down issues when there are flow errors.  The following should work:
```
function generateChoice(options: $Partial<ChoiceType>): ChoiceType {
   ...
}
```